src/panels/config/voice-assistants/dialog-voice-assistant-pipeline-detail.ts

```diff
@@ -279,6 +286,19 @@ export class DialogVoiceAssistantPipelineDetail extends LitElement {
     }
   }
 
+  private _supportedConversationLanguagesChanged(ev) {
+    this._supportedConversationLanguages = ev.detail.value;
+    if (
+      this._data?.conversation_language &&
+      !this._supportedConversationLanguages?.includes(
+        this._data.conversation_engine!
```

Comment:
```suggestion
        this._data.conversation_language!
```
copy/paste error I think

Reply:
But I tested it :laughing: